### PR TITLE
Added program for international RSE day

### PR DESCRIPTION
**Pull request by annefou**
We need to advertise this even asap.

### CODE REVIEW

File: content/events/international-rse-day/_index.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+- All day, at anytime come and say hello in [https://coderefinery.zulipchat.com](https://coderefinery.zulipchat.com) in our [nordic-rse/international RSE day Oct 14 stream](https://coderefinery.zulipchat.com/#narrow/stream/213720-nordic-rse/topic/international.20RSE.20day.20Oct.2014). We do not have any registration form for this event but we would like to encourage our community to come along in zulip to say hello! Once your 1 minute video about yourself is ready, share it in this stream too!
+- 09:00 Introduce yourself (2 hours come along helpdesk session)
+  The goal of this session is to provide support to prepare a 1 mn video about yourself:
+     - your name, institution, RSE project you are working on e.g. what software are you contributing to (language, topic and whatever you can say in less than 1 minute!)
+
+     All the videos will be published in our youtube channel and we will also make a "wall of fame" on the nordic-RSE website. 
+
+     Nordic-RSE staff will provide helpdesk so they can support you to prepare the video but of course, feel free to skip this session and send us directly the link to your 1 minute video!
+
+- 11:00 Lunch
```
Comment (samumantha) [inline content/events/international-rse-day/_index.md:28]:
```suggestion
- 11:00 Lunch break 
```

File: content/events/international-rse-day/_index.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+- 13:00 TIL session
```
Comment (samumantha) [inline content/events/international-rse-day/_index.md:29]:
```suggestion
- 13:00 TIL ('Today I Learned')  session
```

File: content/events/international-rse-day/_index.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+     - We have many topics covered in zulip. The idea is to write blogs/documents them on the Nordic-RSE website.
```
Comment (samumantha) [inline content/events/international-rse-day/_index.md:30]:
```suggestion
     - This session is all about sharing the small things that we learn now and then, that others may also find useful.
     - You can find a growing collection of TIL on our Zulip #help/TIL channel that can serve as inspiration. 
     - The idea for this afternoon is to write blogs/documents about them to share them with the community via the Nordic-RSE website.
```

**Review by samumantha — APPROVED** [review]
A few suggestions to make it a bit clearer what is happenening (?)
Feel free to ignore if it does not help. Thanks for organizing, Anne :)

**Comment by samumantha** [comment]
Will you also notify the event to be listed here: https://researchsoftware.org/council/intl-rse-day.html ?

**Comment by bast** [comment]
Thank you for the initiative and the great ideas!

**Pull request merged**